release/win/dataloader.bat

```diff
@@ -49,7 +49,7 @@ echo.
     )
 
 :Run
-    java -jar %DATALOADER_UBER_JAR_NAME% salesforce.config.dir=configs
+    java -jar %DATALOADER_UBER_JAR_NAME% %*
```

Comment:
As synonymous to what's done in `dataloader.command`, is `%*` how Windows does it?